clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerTestBuilder.java

```
@@ -279,7 +287,7 @@ public ConsumerNetworkThreadTestBuilder(Optional<GroupInformation> groupInfo) {
 
         @Override
         public void close() {
-            closeQuietly(consumerNetworkThread, ConsumerNetworkThread.class.getSimpleName());
+            consumerNetworkThread.close();
```

I don't think we should suppress the failures on closing during testing.